x/dispensation/Steps-AuthorizedRunner.md

```diff
@@ -0,0 +1,200 @@
+# Steps to follow by Authorized Runner
+
+## Requirements
+1. A dispensation should already be created : A dispensation should have been created, and the authorized runner address assigned to it.
+2. Enough funds to pay for gas fees : The authorized runner would need to pay for the gas fee everytime they run a new transaction.
```

Comment:
2. The authorized runner would need to pay for the gas fee every time they run a new transaction.